packages/vuepress-theme-odyssey/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@okta/vuepress-theme-odyssey",
+  "version": "0.6.0",
+  "description": "VuePress theme for Odyssey Documentation",
+  "author": "Okta, Inc.",
+  "license": "Apache-2.0",
+  "private": true,
+  "main": "index.js",
+  "devDependencies": {
+    "@okta/odyssey": "^0.6.0",
+    "@okta/odyssey-icons": "^0.6.0",
+    "html-loader": "^1.1.0",
+    "node-sass": "4.13.1",
+    "sass-loader": "8.0.2",
+    "stylelint": "^9.3.0",
+    "stylelint-checkstyle-formatter": "^0.1.1",
+    "stylelint-config-recommended": "^2.1.0",
+    "stylelint-config-sass-guidelines": "^5.0.0",
+    "stylelint-config-standard": "^18.2.0",
+    "stylelint-order": "^0.8.1",
+    "stylelint-scss": "^3.2.0",
+    "vue-fragment": "^1.5.1",
+    "vuepress-plugin-clean-urls": "^1.1.1"
```

Comment:
Please double check that dependencies are correctly specified as `devDependencies` or `dependencies`. [Here is a detailed explanation](https://stackoverflow.com/a/22004559/7059401) of the differences between the two.

Reply:
Done